bcda/logging/middleware.go

```diff
@@ -83,6 +86,48 @@ func (l *StructuredLoggerEntry) Panic(v interface{}, stack []byte) {
 	})
 }
 
+type ResourceTypeLogger struct {
+	Repository models.JobKeyRepository
+}
+
+func (rl *ResourceTypeLogger) LogJobResourceType(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		next.ServeHTTP(w, r)
```

Comment:
Am I reading this correctly that only a 200 response from `v1.ServeData` will result in the resource type being logged?  Not necessarily a bad thing, just wanted to make sure I was interpreting this middleware function correctly.

Reply:
This should run regardless, although if ServeData isn't returning a 200 it's likely that the same thing that caused that will mean there's no ResourceType to log (e.g. the job id was wrong, or something)